src/pages/eunjungPark/Login/Login.js

```diff
@@ -1,8 +1,114 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Login.scss';
 
 class LoginEunjung extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  loginButton = () => {
+    const { id, pw } = this.state;
+    const checkId = id.includes('@');
+    const checkPw = pw.length >= 4;
+
+    if (checkId && checkPw) {
+      alert('입력성공');
+
+      fetch('http://10.58.3.176:8000/users/signin', {
+        method: 'POST',
+        body: JSON.stringify({
+          name: '이름',
+          email: this.state.id,
+          password: this.state.pw,
```

Comment:
- 위에서 구조분해할당 해주셨으니, 활용해주세요!
```suggestion
          email: id,
          password: pw,
```